pyaedt/hfss.py

```diff
@@ -550,7 +537,8 @@ def create_linear_step_sweep(self, setupname, unit, freqstart, freqstop, step_si
         <class 'pyaedt.modules.SetupTemplates.SweepHFSS'>
 
         """
-
+        if sweep_type not in ["Discrete", "Interpolating", "Fast"]:
+            self.add_error_message("Error in Sweep Type. it has to be Discrete, Interpolating or Fast")
```

Comment:
```suggestion
            self.add_error_message("Invalid in `sweep_type`. It has to either 'Discrete', 'Interpolating', or 'Fast'")
```
Seems like it would be best to raise a ValueError rather than to let this silently pass in Python.